boards/common/iotlab/include/periph_conf_common.h

```diff
@@ -74,16 +74,12 @@ extern "C" {
  */
 #ifdef MODULE_PERIPH_DMA
 static const dma_conf_t dma_config[] = {
-    { .stream = 1 },    /* DMA1 Channel 2 - SPI1_RX */
```

Comment:
You could keep this, even though it's not used. I would just wast a couple of bytes in flash.